lib/ruby/lib/alces/stack/kickstart/run.rb

```diff
@@ -44,36 +45,36 @@ def initialize(template, options={})
         end
 
         def run!
+          raise "Ran from boot can only run with a single node" if @ran_from_boot and @group
           if @dry_run_flag
-            lambda = -> (json) { puts_template(json) }
+            lambda = -> (template_parameters) { puts_template(template_parameters) }
```

Comment:
I'd avoid calling the variable `lambda` as it's also a Ruby keyword.  A good alternative would be to describe what the lambda does, say, `iterator_proc`.